dhis-2/dhis-api/src/test/java/org/hisp/dhis/program/ProgramStageTest.java

```diff
@@ -129,7 +129,7 @@ void testCopyOfCodeShouldBeNullWhenOriginalHasNullCode() {
   @Test
   void testExpectedFieldCount() {
     Field[] allClassFieldsIncludingInherited = getAllFields(ProgramStage.class);
-    assertEquals(50, allClassFieldsIncludingInherited.length);
+    assertEquals(52, allClassFieldsIncludingInherited.length);
```

Comment:
is there another way of getting this value instead of hard coding 52 as the expected one?
seems a bit arbitrary, and personally, when I read the test I wouldn't know how to verify if 52 is the expected value
same goes for the other tests with hard coded expected field count values